runtime/server/column_api_test.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"github.com/rilldata/rill/runtime/api"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestServer_GetTopK(t *testing.T) {
+	server, instanceId, err := GetTestServer()
+
+	server.QueryDirect(context.Background(), &api.QueryDirectRequest{
+		InstanceId: instanceId,
+		Sql:        "CREATE TABLE test AS (SELECT 'abc' AS col, 1 AS val UNION ALL SELECT 'def' AS col, 5 AS val UNION ALL SELECT 'abc' AS col, 3 AS val)",
+		Args:       nil,
+	})
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	if server == nil {
+		t.Fatal("server is nil")
+	}
+	if instanceId == "" {
+		t.Fatal("instanceId is empty")
+	}
```

Comment:
These checks should go after `getTestServer` – but actually, since it's a helper function, you could just make it `getTestServer(t)`, and add the assertions directly in there.

Also, since we're using the `require` package, you can just do `require.NoError(t, err)` or `require.NotNil(t, server)` for these kinds of checks

Reply:
done